script/gen-types.ts

```diff
@@ -233,11 +233,7 @@ const out = [
                         typeAnnotation: b.tsTypeLiteral.from({
                           members: Object.keys(typeDef.allFields)
                             .filter(fieldName => fieldName !== "type")
-                            .sort((fieldName1, fieldName2) => {
-                              const field1 = typeDef.allFields[fieldName1];
-                              const field2 = typeDef.allFields[fieldName2];
-                              return field1.name < field2.name ? -1 : 1;
-                            })
+                            .sort() // Sort field name strings lexicographically.
```

Comment:
Huh. The previous code doesn't make any sense. Why use the `fieldName` to get the `field.name`... Wonder if that was left over from a refactor. Thanks for cleaning that up.